00-Introducing-HOOMD-blue/06-Equilibrating-the-System.ipynb

```diff
@@ -32,6 +32,20 @@
     "import math"
```

Comment:
<p>I agree with having xb here. I think it would be useful to at least state what xb means somewhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>